src/databricks/labs/ucx/hive_metastore/locations.py

```diff
@@ -160,7 +160,7 @@ def _get_ext_location_definitions(self, missing_locations: list[ExternalLocation
             cnt += 1
         return tf_script
 
-    def _match_table_external_locations(self) -> tuple[list[list], list[ExternalLocation]]:
+    def match_table_external_locations(self) -> tuple[list[list], list[ExternalLocation]]:
```

Comment:
Return type is incorrect. Add more unit tests for this method if you make it public